pkgs/top-level/aliases.nix

```diff
@@ -597,6 +597,7 @@ mapAliases ({
   nix_2_4 = nixVersions.nix_2_4;
   nix_2_5 = nixVersions.nix_2_5;
   nix_2_6 = nixVersions.nix_2_6;
+  nixops = nixops_unstable;
```

Comment:
Should be an error instead, otherwise we'd be updating all nixops users to unstable eithout warning.

But then also it would be more proper to just deprecate stable nixops softly with a warning first for a release.

Reply:
It's been failing to eval for months now due to Python 2 insecure stuff.

Reply:
Updated to a throw.

Reply:
Is it usable in the stable release? If so, we need to deprecate it with a warning or error

Reply:
It's not, Python 2.7 is marked insecure in 23.05 as well.

Reply:
Wouldn't somebody running into that just ignore the warning?

Reply:
I would hope not, given 2.7 was EOL in _2020_.

Reply:
Still, let's have at least an error here. People should be made aware that they're using the stable attribute and should switch to the unstable one

Reply:
I did make it a throw.